gunicorn/instrument/statsd.py

```diff
@@ -99,21 +99,21 @@ def access(self, resp, req, environ, request_time):
             status = int(status.split(None, 1)[0])
         self.histogram("gunicorn.request.duration", duration_in_ms)
         self.increment("gunicorn.requests", 1)
-        self.increment("gunicorn.request.status.%d" % status, 1)
+        self.increment(f"gunicorn.request.status.{status}", 1)
```

Comment:
Even though the `status` variable _should_ have an integer type here, let's maintain the existing integer presentation type for it during string formatting.